packages/libs/codemodel/.resources/all-in-one/json/code-model.json

```diff
@@ -880,6 +880,13 @@
             "$ref": "#/definitions/Parameter"
           }
         },
+        "specialHeaders": {
+          "description": "List of the headers that the generator should automatically handle.",
```

Comment:
Proposal: List of headers that parameters should not handle as parameters but with special logic

Reply:
Some aka.ms link to some autorest doc?